installing/installing_azure/ipi/installing-azure-default.adoc

```diff
@@ -1,13 +1,12 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="installing-azure-default"]
-= Installing a cluster quickly on Azure
+= Installing a cluster on {azure-short}
 include::_attributes/common-attributes.adoc[]
```

Comment:
If you use a variable in the title of the first assembly in a section, move the include attributes directive above the anchor ID in this assembly. Otherwise, the variable will not render correctly on our documentation portals.

Reply:
done. Since these are part of common refactoring effort across platform that are already PM approved, we do not require QE approval.